packages/storybook/src/executors/storybook/schema.json

```diff
@@ -89,6 +89,11 @@
       "type": "boolean",
       "description": "Suppress verbose build output.",
       "default": true
+    },
+    "useDefaultProject": {
+      "type": "boolean",
+      "description": "Will use the default project as specified in the workspace configuration.",
+      "default": false
```

Comment:
Wouldn't this be a breaking change though? I think most of the current projects (also at clients) rely on the "default project resolution" that Storybook had up until 6.4. 

The implementation for resolving the default within the Nx Storybook executor has been added to make sure that behavior keeps being there and doesn't break existing Nx Storybook projects when upgrading to 6.4.

Another thing that might be problematic is what happens if someone specifies

```
{
   useDefaultProject: true,
   projectName: 'some-project-different-from-default'
}
```

These are mutually exclusive, right? Which one would take precedence in this case?